modules/ROOT/examples/deployment/container/orchestration/nginx-ingress-tab-1.yaml

```diff
@@ -1,29 +1,28 @@
-externalDomain: ocis.owncloud.test
+externalDomain: ocis.kube.owncloud.test
```

Comment:
At the end we are adding `ocis.kube.owncloud.test` to the `/etc/hosts`. So, I thought to make it consistent everywhere.